src/h5web/visualizations/shared/VisCanvas.tsx

```diff
@@ -9,7 +9,7 @@ import AxisSystemProvider from './AxisSystemProvider';
 
 const AXIS_OFFSETS = { vertical: 72, horizontal: 36, fallback: 16 };
 
-interface Props {
+export interface Props {
```

Comment:
Should we rename `Props` to `VisCanvasProps` avoid exporting an interface with a too generic name ?

Reply:
You're right, I put 

```js
export type { Props as VisCanvasProps };
```

at the bottom of the file instead, just before the default export.

Reply:
This allows us to keep the simple name `Props` inside the file.